susemanager-utils/testing/docker/scripts/schema_idempotency_test_pgsql.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import difflib
+import distutils.version as DV
+import json
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+
+def file_exists(path):
+    try:
+        open(path, 'r')
+        return(True)
+    except FileNotFoundError:
+        return(False)
+
+
+def directory_exists(path):
```

Comment:
I would suggest to simply inline `os.path.isdir(path)` and remove this function.

If you are tired of typing dots (which is something I can fully understand) you can also import the isdir/isfile symbols directly.

Reply:
Same as before.

Fixed and testing.